src/scss/core.scss

```diff
@@ -33,5 +33,16 @@
   filter: alpha(opacity=0);
 }
 
+.select2-hidden-accessible {
+    border: 0;
+    clip: rect(0 0 0 0);
+    /*height: 0px;*/
```

Comment:
jQuery UI uses `height: 1px` and `width: 1px` to work around this.

https://github.com/jquery/jquery-ui/blob/0de27b0609e8f6f9751ab7cce28492e18206d86d/themes/base/core.css#L17-L26